src/istore.c

```diff
@@ -976,3 +976,73 @@ istore_svals(PG_FUNCTION_ARGS)
 
     SRF_RETURN_DONE(funcctx);
 }
+
+/*
+ * A boolean function that supports `select '1=>1'::istore = '2=>2'::istore`
+ */
+PG_FUNCTION_INFO_V1(istore_eq);
+Datum
+istore_eq(PG_FUNCTION_ARGS)
+{
+    IStore *is1, *is2;
+    IStorePair *pairs1, *pairs2;
+    int32 len;
+
+    if (PG_ARGISNULL(0) || PG_ARGISNULL(1)) PG_RETURN_NULL();
```

Comment:
that's true but the STRICT is defined on the SQL level and you never know in which context the function is calles (imagine `DirectFunctionCall2`) so we should prefer being save than sorry